index.js

```diff
@@ -310,5 +310,13 @@ client.on("messageCreate", async (message) => {
 client.on("error", async (error) => {
   console.log("❌ | An error has ocurred", error);
 });
-
+/* aportación del ruso */
+
+client.on("messageCreate". (quebueno) => {

```

Comment:
Implement this code in the current `messageCreate` event handler and I'll merge

Reply:
Also needs a way to work only in a specific guild

Reply:
fill de pute ta mere la claustra